### PR TITLE
Issue #42: Documentation Section by Chris & Freddie

### PR DESCRIPTION
**Pull request by christopherchateau**
# Pull Request Template

## Description: Create index.html and styles.css

This PR addresses issue #42

The changes made to the codebase in this PR:

* Index.html was created to walk users through using the API endpoints
* Styling was added from styles.css

## Requests for review

Areas of concern:

* Spelling and correct instructions were double checked, but an additional look is always a good idea.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+              "message": "Station \"Chris' awesome station\" successfully created!"
+            }
+              </code></pre>
+
+        <h3 class="endpoint-title">Put Endpoints</h3>
+        <p class="instructions">
+          To change the name of a specific charging station, use the URL:
+        </p>
+        <p class="url">
+          https://recharge-api.herokuapp.com/api/v1/stations/STATIONIDHERE
```
Comment (alexanderela) [inline public/index.html]:
```suggestion
          https://recharge-api.herokuapp.com/api/v1/stations/:station_id
```

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,384 @@
...
+            } 
+          </code></pre>
+        <h4 class="example">Example Response:</h4>
+        <pre><code>
+            {
+              "message": "Edit successful. Station with id of 35 name changed from Chris' awesome station to Freddie's awesome station."
+            } 
+          </code></pre>
+
+        <h3 class="endpoint-title">Delete Endpoints</h3>
```
Comment (alexanderela) [inline public/index.html]:
```suggestion
        <h3 class="endpoint-title">DELETE Endpoints</h3>
```

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,384 @@
...
+        <p class="instructions">
+          To delete a specific charging station, use the URL:
+        </p>
+        <p class="url">
+          https://recharge-api.herokuapp.com/api/v1/stations/STATIONIDHERE
```
Comment (alexanderela) [inline public/index.html]:
```suggestion
          https://recharge-api.herokuapp.com/api/v1/stations/:station_id
```

File: public/index.html [hunk 4]
```diff
@@ -0,0 +1,384 @@
...
+              "message": "Station 33 has been deleted."
+            }
+          </code></pre>
+      </section>
+    </article>
+
+    <article class="cafe-article">
+      <h2 class="cafe-title">Cafe Endpoints</h2>
+      <section class="cafe-endpoints">
+        <h3 class="endpoint-title">Get Endpoints</h3>
```
Comment (alexanderela) [inline public/index.html]:
```suggestion
        <h3 class="endpoint-title">GET Endpoints</h3>
```

File: public/index.html [hunk 5]
```diff
@@ -0,0 +1,384 @@
...
+              "updated_at": "2018-12-11T01:10:17.833Z"
+            }
+          ]
+          </code></pre>
+
+        <p class="instructions">
+          To get data for a specific cafe, use the URL:
+        </p>
+        <p class="url">
+          https://recharge-api.herokuapp.com/api/v1/stations/STATIONIDHERE/cafes
```
Comment (alexanderela) [inline public/index.html]:
```suggestion
          https://recharge-api.herokuapp.com/api/v1/stations/:station_id/cafes
```

File: public/index.html [hunk 6]
```diff
@@ -0,0 +1,384 @@
...
+              "station_id": 1,
+              "created_at": "2018-12-11T01:10:17.834Z",
+              "updated_at": "2018-12-11T01:10:17.834Z"
+            }
+          ]
+          </code></pre>
+
+        <p class="instructions">To get a specific cafe by id, use the URL:</p>
+        <p class="url">
+          https://recharge-api.herokuapp.com/api/v1/stations/STATIONIDHERE/cafes/CAFEIDHERE
```
Comment (alexanderela) [inline public/index.html]:
```suggestion
          https://recharge-api.herokuapp.com/api/v1/stations/:station_id/cafes/:cafe_id
```

File: public/index.html [hunk 7]
```diff
@@ -0,0 +1,384 @@
...
+                "formatted_address": "1550 Court Pl (Sheraton Denver Downtown), Denver, CO 80202, United States",
+                "distance_in_meters": 368,
+                "station_id": 1,
+                "created_at": "2018-12-11T01:10:17.835Z",
+                "updated_at": "2018-12-11T01:10:17.835Z"
+              }
+            ]
+            </code></pre>
+
+        <h3 class="endpoint-title">Post Endpoints</h3>
```
Comment (alexanderela) [inline public/index.html]:
```suggestion
        <h3 class="endpoint-title">POST Endpoints</h3>
```

File: public/index.html [hunk 8]
```diff
@@ -0,0 +1,384 @@
...
+        <p class="instructions">To add a cafe to the database, use the URL:</p>
+        <p class="url">
+          https://recharge-api.herokuapp.com/api/v1/stations/STATIONIDHERE/cafes
```
Comment (alexanderela) [inline public/index.html]:
```suggestion
          https://recharge-api.herokuapp.com/api/v1/stations/:station_id/cafes
```

File: public/index.html [hunk 9]
```diff
@@ -0,0 +1,384 @@
...
+              </code></pre>
+        <h4 class="example">Example Response:</h4>
+        <pre><code>
+            {
+              "id": 103,
+              "message": "Cafe "Chris' awesome cafe" successfully created!"
+            }
+                </code></pre>
+
+        <h3 class="endpoint-title">Put Endpoints</h3>
```
Comment (alexanderela) [inline public/index.html]:
```suggestion
        <h3 class="endpoint-title">PUT Endpoints</h3>
```

File: public/index.html [hunk 10]
```diff
@@ -0,0 +1,384 @@
...
+        <p class="instructions">
+          To change the name of a specific cafe, use the URL:
+        </p>
+        <p class="url">
+          https://recharge-api.herokuapp.com/api/v1/stations/STATIONIDHERE/cafes/CAFEIDHERE
```
Comment (alexanderela) [inline public/index.html]:
```suggestion
          https://recharge-api.herokuapp.com/api/v1/stations/:station_id/cafes/:cafe_id
```

File: public/index.html [hunk 11]
```diff
@@ -0,0 +1,384 @@
...
+            </code></pre>
+        <h4 class="example">Example Response:</h4>
+        <pre><code>
+            {
+              "cafe": 5,
+              "message": "Edit successful. Cafe with id of 2 name changed from Peet's Coffee & Tea to Freddie's awesome cafe."
+            } 
+            </code></pre>
+
+        <h3 class="endpoint-title">Delete Endpoints</h3>
```
Comment (alexanderela) [inline public/index.html]:
```suggestion
        <h3 class="endpoint-title">DELETE Endpoints</h3>
```

File: public/index.html [hunk 12]
```diff
@@ -0,0 +1,384 @@
...
+        <p class="instructions">
+          To change the name of a specific cafe, use the URL:
+        </p>
+        <p class="url">
+          https://recharge-api.herokuapp.com/api/v1/stations/STATIONIDHERE/cafes/CAFEIDHERE
+        </p>
+        <p class="instructions">
+          in your fetch request, which returns a response body with an id and a
+          message.
+        </p>
```
Comment (alexanderela) [inline public/index.html]:
```suggestion
        </p>
        <pre><code>
            https://recharge-api.herokuapp.com/api/v1/stations/1/cafes/2
              </code></pre>
```

File: public/index.html [hunk 13]
```diff
@@ -0,0 +1,384 @@
...
+        <h4 class="example">Example Request:</h4>
+        <pre><code>
+            {
+              "cafe_name": "Freddie's awesome cafe"
+            }
+            </code></pre>
+        <h4 class="example">Example Response:</h4>
+        <pre><code>
+            {
+              "cafe": 5,
```
Comment (alexanderela) [inline public/index.html]:
```suggestion
              "cafe": 2,
```

File: public/index.html [hunk 14]
```diff
@@ -0,0 +1,384 @@
...
+              "message": "Edit successful. Cafe with id of 2 name changed from Peet's Coffee & Tea to Freddie's awesome cafe."
+            } 
+            </code></pre>
+
+        <h3 class="endpoint-title">Delete Endpoints</h3>
+        <p class="instructions">To delete a specific cafe, use the URL:</p>
+        <p class="url">
+          https://recharge-api.herokuapp.com/api/v1/cafes/CAFEIDHERE
```
Comment (alexanderela) [inline public/index.html]:
```suggestion
          https://recharge-api.herokuapp.com/api/v1/cafes/:cafe_id
```

File: public/index.html [hunk 15]
```diff
@@ -0,0 +1,384 @@
...
+          message.
+        </p>
+        <h4 class="example">Example URL:</h4>
+        <pre><code>
+            http://localhost:3000https://recharge-api.herokuapp.com/api/v1/cafes/2
+            </code></pre>
+        <h4 class="example">Example Response:</h4>
+        <pre><code>
+              {
+                "id": 1,
```
Comment (alexanderela) [inline public/index.html]:
```suggestion
                "id": 2,
```

File: public/styles.css [hunk 16]
```diff
@@ -0,0 +1,37 @@
...
+  color: white;
+}
+
+.main-headline {
+  font-size: 36px;
+  text-align: center;
+}
+
+.endpoint-title {
+  font-size: 28px;
```
Comment (colehart) [inline public/styles.css]: Rems, please.
```suggestion
  font-size: 1.75rem;
```

File: public/styles.css [hunk 17]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+
+.charging-station-article,
```
Comment (colehart) [inline public/styles.css]: Refactor to either
```
.article

OR

article
```

File: public/styles.css [hunk 18]
```diff
@@ -0,0 +1,37 @@
...
+.cafe-article {
+  background: rgb(60, 60, 60);
+  border: 4px solid white;
+  margin: 20px;
+  padding: 20px;
+}
+
+.charging-station-title,
+.cafe-title {
+  font-size: 32px;
```
Comment (colehart) [inline public/styles.css]: Rems, please
```suggestion
  font-size: 2rem;
```

**Comment by christopherchateau** [comment]
Updated index.html and styles.css per author requests.

**Pull request merged**